test/full-program-tests/pinned-actor-no-timer/main.pony

```diff
@@ -0,0 +1,36 @@
+use @pony_exitcode[None](code: I32)
+use @usleep[I32](micros: U32) if not windows
+use @Sleep[None](millis: U32) if windows
+use "actor_pinning"
+
```

Comment:
whenever this is "done, can you add comments explaining the various bit that this is testing?

i think it will be non-obvious to many folks even if I understand.

Reply:
from the comment, perhaps we should call this regression-4582 ?

we have a number of full program tests with that naming scheme, named after the regression they are testing against.

Reply:
done